### ISSUE TITLE
training data are being loaded when test script is called

### ISSUE THREAD

**Issue by Omar-Aldade** [hailanyi/VirConv#24]
Hello again,

it's less a code issue and more an understanding issue. I wanted to manipulate the input data, regenerate the dense point cloud with PENet and then test the performance of the pretrained VirConv_L network. When running the test script i've noticed that the "velodyne_depth" data in the "training" folder are being loaded. Shouldn't the data from the "testing" folder be loaded instead?

I was also a bit confused when generating the dataset info that the dataset is devided into three sets, which are training, testing and validation. Can you please explaing to me how the data is used to test the performance as it's hard to follow the code?

 i'm only interestend in the test set and to save time i would like to skip generating the dense point cloud for the training set.

Thank you so much in advance.

**Comment by hailanyi** [comment]
The KITTI dataset did not provided the ground truth of testing dataset. So the performance can not be directly tested on testing dataset. Instead, the performance is generally validated on training dataset. I followed the previous work, that split the training dataset(7481 frames) into a train set (3712 frames) and a validation set (3769 frames). The detector is only trained on train set (3712 frames). The test script tests the performance on the validation set (3769 frames). So the script loads the data from training set. You need to submit your detection results to KITTI online server if you want to test the performance on testing set.

**Comment by Omar-Aldade** [comment]
Thank you for the explanation :)
